zzz_generated/all-clusters-app/zap-generated/endpoint_config.h

```diff
@@ -97,39 +97,39 @@
                                                                                                                                    \
             /* 1016 - VendorName, */                                                                                               \
             0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,      \
-            0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,                                          \
+            0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,                                    \
```

Comment:
Why do each of these have an extra byte in them? Is this a null terminator space? Is this expected?

Reply:
These have an extra byte because this zap update picked a change to zap made by @vivien-apple , that deals with an extra byte to the string types.

@vivien-apple : I asked above, and I'm not sure myself what exactly you were fixing there. Can you elaborate, please?